storage/ndb/rest-server/rest-api-server/internal/integrationtests/ping/performance_test.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * This file is part of the RonDB REST API Server
+ * Copyright (c) 2023 Hopsworks AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ping
+
+import (
+	"sort"
+	"testing"
+	"time"
+
+	"hopsworks.ai/rdrs/internal/config"
+	"hopsworks.ai/rdrs/internal/testutils"
+)
+
+/*
+	The number of parallel client go-routines spawned can be influenced by setting
+	runtime.GOMAXPROCS(). It defaults to the number of CPUs.
+
+	go test \
+		-test.bench BenchmarkSimple \
+		-test.run=thisexpressionwontmatchanytest \
+		-cpu 1,2,4,8 \
```

Comment:
does -cpu 1,2,4,8 has any impact? 
sometimes you override this with
runtime.GOMAXPROCS(runtime.NumCPU() * 5)

Reply:
Haven't used it so far!